src/components/AdminPanelLayout/UsersPane.tsx

```diff
@@ -0,0 +1,48 @@
+import {Table} from 'react-bootstrap';
+import React, {useState} from 'react';
+import {User} from '../../types/User';
+import {getUserList} from '../../api';
+import {useMount} from '../../hooks/useMount';
+import {HorizontallyCenteredSpinner} from '../util/HorizonallyCenteredSpinner';
+import {Layout} from '../../pages/Layout/Layout';
+
+
+const UsersPane = () => {
+  const [users, setUsers] = useState();
+  const [loading, setLoading] = useState(true);
+
+  useMount(async () => {
+    setUsers(await getUserList());
+    completeLoading();
+  });
+
+  function completeLoading() {
+    setLoading(false);
+  }
+
+  const renderUserList = () => (
```

Comment:
I think it's fine to inline this function implementation too. It's only called in one place. It's easier to see top level logic like `loading ? <HorizontallyCenteredSpinner/> :` if it's inlined in the ternary else expression.

Reply:
This is giving me some trouble. For some reason React is convinced that this isn't a valid component to render when I inline the function. Calling it as a named function works fine, but when I inline it, React refuses to render it.

![image](https://user-images.githubusercontent.com/15639276/123728844-fd6ae300-d861-11eb-983e-c36021290708.png)

Reply:
Updated this